### DISCUSSION TITLE
I need to change modifier, what are some good alternatives ?

### DISCUSSION THREAD

**Discussion by BenoitAverty** [i3/i3#5268] [Q&A] [answered]
Hello,

I've been using i3 for several years with the windows key as my modifier. However, this caused me to form the habit of having my thumb constantly bent under my hand, over the windows key, and this bad habit in turn is causing me a tendinitis of the thumb.

I need to change my habit : either changing my muscle memory to use my index finger instead of my thumb to press the modifier (that seems very difficult) or changing my modifier key to force me to change habits.

That task seems daunting but I don't have any choice. So I'm looking for the best alternative. I'm a heavy keyboard user so I use keyboard shortcuts in most of my daily tasks (as a developer). It seems to me that the windows key is the only one that don't have multiple conflicts with other shortcuts. I'm trying the AltGr key at the moment but it's preventing me to type some characters on my french keyboard. I'm thinking of moving the altGr key to the windows key but then it'll be a pain to use another keyboard or another computer...

I'm looking for people who use something else than the windows key : what modifier have you chosen ? Why ? Did you need to change other default shortcuts ?

Thanks in advance

**Comment by okraits** [comment, marked as answer]
I remap capslock to Mod4 since many years - a habit back from the time when thinkpads didn't have a windows key. It's easy to reach capslock with the pinky.

Might that be a solution for you?

**Reply by BenoitAverty** [reply]
Is that compatible with using caps lock or do you have caps lock remapped somewhere else ? I think that would fix may pain problems so that's definitely worth a try, thanks !

**Reply by okraits** [reply]
No, I haven't. I don't need the original capslock function. You're welcome :grin:

**Comment by EgZvor** [comment]
Here's my layout, using super as a modifier

![660_pd](https://user-images.githubusercontent.com/11237935/202189054-678d01ac-0ab7-40ed-a4f9-a8f418732833.png)
